libs/components/router/src/lib/modules/href/fixtures/href-fixture.component.ts

```diff
@@ -13,31 +13,35 @@ import {
 export class HrefDirectiveFixtureComponent {
   @Input()
   public set dynamicLink(value: string | any[]) {
-    this._dynamicLink = value;
-    this.changeDetectorRef.markForCheck();
+    this.#_dynamicLink = value;
+    this.#changeDetectorRef.markForCheck();
```

Comment:
I attempted to remove it and it caused tests to fail. As this is a testing fixture component I'm inclined to just leave it instead of spending cycles trying to figure it out just for a unit test.